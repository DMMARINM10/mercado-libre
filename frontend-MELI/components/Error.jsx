const Error = () => {
  return (
    <p className="error">
        ¡Algo salió mal!
        <br /> 
        {':('}
    </p>
  )
}

export default Error
