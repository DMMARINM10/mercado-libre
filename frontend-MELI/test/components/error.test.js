import { render } from '@testing-library/react';
import Error from '@/components/Error';

describe('Error component', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Error />);
    expect(asFragment()).toMatchSnapshot();
  });
});