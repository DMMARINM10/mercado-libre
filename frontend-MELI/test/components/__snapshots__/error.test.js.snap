// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should match the snapshot 1`] = `
<DocumentFragment>
  <p
    class="error"
  >
    ¡Algo salió mal!
    <br />
    :(
  </p>
</DocumentFragment>
`;
